src/utils/application.data.ts

```diff
@@ -118,6 +118,18 @@ export const getFromApplicationData = (req: Request, key: string, id: string, er
   }
 };
 
+export const getCountOfBoWithTrusteeNoc = (session: Session | undefined) => {
```

Comment:
Not sure we need the actual count. Consider using return (.some() || .some())

Reply:
Good point - updated to use `some` instead of `filter`.